### PR TITLE
GFM-51 - Fix Acceptance Tests

### PR DESCRIPTION
**Pull request by joefitter**
* Added missing dependency - i18n-lookup - this was removed from HOF
* Reference i18n-lookup directly from emailer
* Fixed broken acceptance test - "Happy path through details, then back through type" - If you complete 'details' page (and it isn't invalidated) then you shouldn't be taken to the additional page - this step was added in error.
* remove /edit check from fork conditionals - this is not needed with step invalidation
* bumped hof to 8.0.0

### CODE REVIEW

**Comment by joefitter** [comment]
@gxxm @bendiggle @JoeChapman happy to merge this? NPM > 3 needs to be used with hof > 7.0.1 as per discussion earlier

**Comment by joefitter** [comment]
@JoeChapman @gxxm @bendiggle this one can be merged now, the travis config was moved to another branch

File: package.json [hunk 1]
```diff
@@ -40,9 +40,10 @@
     "express": "^4.12.4",
     "express-partial-templates": "^0.1.0",
     "express-session": "^1.11.3",
-    "hof": "^1.2.0",
+    "hof": "^8.0.0",
```
Comment (JoeChapman) [inline package.json]: lol. How come this is soooo out of date? (Rhetorical question) Amaze balls

File: services/email/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ var nodemailer = require('nodemailer');
 var config = require('../../config');
 var i18n = require('hof').i18n;
 var Hogan = require('hogan.js');
-var i18nLookup = require('hof').i18nLookup;
+var i18nLookup = require('i18n-lookup');
```
Comment (JoeChapman) [inline services/email/index.js]: Is this because `require('hof').i18nLookup` is out of date?

File: package.json (hunk 1 above)
Comment (joefitter) [inline package.json]: I have no idea, we tried bumping it to 7.1 before but broke on node < 3 because of reasons discussed earlier

Comment (joefitter) [inline package.json]: example form etc will probably be using an ooold version too

File: services/email/index.js (hunk 2 above)
Comment (joefitter) [inline services/email/index.js]: no, it was removed from hof core

Comment (joefitter) [inline services/email/index.js]: about 7 major versions ago ;)

**Pull request merged**